installing/installing_openstack/installing-openstack-installer-sr-iov.adoc

```diff
@@ -62,8 +62,7 @@ include::modules/cluster-telemetry.adoc[leveloffset=+1]
 == Next steps
 
```

Comment:
Use unique ID for "Next steps" titles in assemblies. Refer to the guidance on [Anchoring "Prerequisites", "Additional resources", and "Next steps" titles in assemblies](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies)